@uportal/portlet-registry-to-array/src/portlet-registry-to-array.js

```diff
@@ -9,13 +11,23 @@ function flatten(acc, arr) {
 }
 
 /**
- * Walks the portlet registry tree
+ * Takes the returned array from treeWalker and removes duplicates
+ * baed on "fname"
```

Comment:
While `fname` may be bae, I think based is the word you're looking for here.

Reply:
lol, good catch